CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,122 @@
+# Contributor Covenant Code of Conduct for the [`Canada.ca Search User Interface (UI)`] project
+
+([Français](#code-de-conduite-pour-le-projet-iu-recherche))
+
+Contributors to repositories hosted in [`Canada.ca Search UI`] are expected to follow the Contributor Covenant Code of Conduct, and those working within Government are also expected to follow the Values and Ethics Code for the Public Sector
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the department
+* Showing empathy towards other members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project, members or [`Employment and Social Development Canada`].
+Examples of representing a project, members or [`Employment and Social Development Canada`] include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
```

Comment:
I think the intend here was to remove the bracket + the code markup because they were only indicating a placeholder. If it is to put emphasis, then a emphasis markdown marker should be used instead.

```suggestion
This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project, members or Employment and Social Development Canada.
Examples of representing a project, members or Employment and Social Development Canada include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
```